cmake/external/pyxir.cmake

```diff
@@ -3,8 +3,8 @@
 
 set(PYXIR_SHARED_LIB libpyxir.so)
 
-if(PYTHONINTERP_FOUND)
-  execute_process(COMMAND "${PYTHON_EXECUTABLE}" "-c"
+if(Python_EXECUTABLE)
```

Comment:
would Python_Interpreter_FOUND work as well?

Reply:
Yes, I think that would work too. I am using `Python_EXECUTABLE` because that's used further down in the file. The previous code wouldn't have gotten broken if we hadn't used `PYTHONINTERP_FOUND` and `PYTHON_EXECUTABLE`. The former wasn't set anymore but the latter actually still was when I tested it. That's why I changed it to check the same variable that's used later on now. Any particular reason why you would prefer `Python_Interpreter_FOUND`?

Reply:
I don't have a strong preference. I thought Python_Interpreter_FOUND aligned better with the original check but Python_EXECUTABLE should be fine.